pkg/server/server_sql.go

```diff
@@ -1519,6 +1519,9 @@ func (s *SQLServer) preStart(
 	)
 
 	scheduledlogging.Start(ctx, stopper, s.execCfg.DB, s.execCfg.Settings, s.internalExecutor, s.execCfg.CaptureIndexUsageStatsKnobs)
+	if err := stopper.RunAsyncTask(ctx, "warm-synthetic-privilege-cache", s.execCfg.WarmSyntheticPrivilegeCacheForVirtualTables); err != nil {
```

Comment:
I don't think this is necessarily safe because you're incrementing the waitgroup counter in a goroutine. Doesn't the waitgroup not like it when you add after calls to Wait?